fabric-gamerules-v1/src/main/java/net/fabricmc/fabric/api/gamerule/GameRuleRegistry.java

```diff
@@ -0,0 +1,7 @@
+package net.fabricmc.fabric.api.gamerule;
+
+import net.fabricmc.fabric.impl.gamerule.GameRuleRegistryImpl;
+
+public interface GameRuleRegistry {
+	final GameRuleRegistry INSTANCE = new GameRuleRegistryImpl();
```

Comment:
no need for `final`

Reply:
Other API interfaces have their instance marked as final so I'm gonna keep with that

Reply:
Sure, though all interface fields are `public static final` by default, it doesn't hurt to keep `final`.